### PR TITLE
[MTDSA-4598] Scala 2.12 / SBT 1.3.7

### PR DESCRIPTION
**Pull request by Nboaram**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-  )
+
   .settings(
     resolvers += Resolver.bintrayRepo("hmrc", "releases"),
     resolvers += Resolver.jcenterRepo
   )
+
+libraryDependencies ++= Seq(
+  "uk.gov.hmrc"   %% "bootstrap-play-26" % "1.3.0",
+  "org.pegdown" % "pegdown" % "1.6.0" % "test",
```
Comment (BetaDraconis) [inline build.sbt]: I'm not convinced this is required.

File: build.sbt [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+  "org.scalatest" %% "scalatest" % "3.1.0" % "test",
+  "com.vladsch.flexmark"   % "flexmark-all" % "0.35.10"
```
Comment (BetaDraconis) [inline build.sbt]: This either.

File: build.sbt [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
+scalaVersion := "2.12.10"
+
 lazy val microservice = Project(appName, file("."))
-  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
+  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
   .settings(
-    libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
     evictionWarningOptions in update := EvictionWarningOptions.default.withWarnScalaVersionEviction(false),
     majorVersion := 0
+
```
Comment (BetaDraconis) [inline build.sbt]: This line is not necessary.

**Comment by BetaDraconis** [comment]
License headers don't appear to be working properly on this repo.

File: build.sbt (hunk 1 above)
Comment (Nboaram) [reply build.sbt]: I have removed these two

File: build.sbt (hunk 2 above)
Comment (Nboaram) [reply build.sbt]: I think the last two are required.

**Pull request merged**